### PR TITLE
Main Menu with Controls in the Start!

### PR DESCRIPTION
**Pull request by JJWesker10**
Main menu with Controls at the Beginning before the player begins the game -- in accordance with the control description in the documentation.

### CODE REVIEW

**Review by Arsenic-ATG — CHANGES_REQUESTED** [review]
Unfortunately, SDL_TTF doesn't support newline character ("\n") plus `renderer::draw_text ()` uses `TTF_RenderText_Blended` behind the scenes to render text on screen, which also doesn't support text wrapping ( I am planning to fix it in future )

so currently the only way to render a newline is to call `renderer::draw_text ()` again with different co-ordinated (if you want, you can look at `renderer.hpp` or `renderer.cpp` to know more about how `draw_text ()` function works )

File: src/game.cpp [hunk 1]
```diff
@@ -493,10 +493,20 @@ game::draw (renderer &p_renderer) -> void
...
+
+        p_renderer.draw_text("CONTROLS:\n LEFT ARROW -- Move Left \n RIGHT ARROW -- Move Right \n Z or UP ARROW -- Rotate Clockwise \n X or LEFT-CTRL -- Rotate Counterclockwise ",
+            coords(p_renderer.get_width() / 2,
+                p_renderer.get_height() / 2 - 100),
+            0xffffffff);
+
+        p_renderer.draw_text("SPACE -- Hard Drop \n DOWN ARROW -- Soft Drop \n P -- Pause Game \n R  Reset Game",
+            coords(p_renderer.get_width() / 2,
+                p_renderer.get_height() / 2 - 120),
+            0xffffffff);
```
Comment (Arsenic-ATG) [inline src/game.cpp:509]:
```suggestion
          coords center (p_renderer.get_width ()/2, p_renderer.get_height ()/2);

          p_renderer.draw_text("CONTROLS:",
                               coords(center.x - 150,
                                      center.y - 130),
                               0xffffffff);

          p_renderer.draw_text("LEFT/RIGHT ARROW -- Move Left/Right",
                               coords(center.x - 100,
                                      center.y - 100),
                               0xffffffff);

          p_renderer.draw_text("Z or UP ARROW -- Rotate Clockwise",
                               coords(center.x - 100,
                                      center.y - 80),
                               0xffffffff);

          p_renderer.draw_text("X or LEFT-CTRL -- Rotate Counterclockwise ",
                               coords(center.x - 100,
                                      center.y - 60),
                               0xffffffff);

          p_renderer.draw_text("SPACE -- Hard Drop",
                               coords(center.x - 100,
                                      center.y - 40),
                               0xffffffff);

          p_renderer.draw_text("DOWN ARROW -- Soft Drop",
                               coords(center.x - 100,
                                      center.y - 20),
                               0xffffffff);

          p_renderer.draw_text("P -- Pause Game",
                               coords(center.x - 100,
                                      center.y),
                               0xffffffff);

          p_renderer.draw_text("R -- Reset Game",
                               coords(center.x - 100,
                                      center.y + 20),
                               0xffffffff);

          p_renderer.draw_text("Press enter to start",
                               coords(center.x - 100,
                                      center.y + 60),
                               0xffffffff);
```

**Review by Arsenic-ATG — APPROVED** [review]
Thanks For contributing to the repository 🎉 🥇

**Comment by JJWesker10** [comment]
> Thanks For contributing to the repository 🎉 🥇

No problem! This was a good first experience for me! 😄

**Pull request merged**